contrib/http_wasm/filters/http/source/vm.h

```diff
@@ -0,0 +1,123 @@
+#pragma once
+
+#include "contrib/http_wasm/filters/http/source/plugin.h"
+#include "contrib/http_wasm/filters/http/source/context.h"
+#include "contrib/http_wasm/filters/http/source/host/context.h"
+#include "envoy/server/lifecycle_notifier.h"
+#include "host/vm_runtime.h"
+#include "source/common/config/datasource.h"
+#include "source/extensions/common/wasm/stats_handler.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace HttpWasm {
+using Common::Wasm::WasmEvent;
+using Host::FailState;
+
+class WasmHandle;
+
+class Wasm : public Host::WasmBase, Logger::Loggable<Logger::Id::wasm> {
```

Comment:
`Wasm` is from the existing code, but I really hate this naming (from dumbass Googlers), we must be really explicit about wtf it's doing in its name. So in the cleaning up phase, let's make it clear for other contributors